pipelines/Latiss/cpSky.yaml

```diff
@@ -10,3 +10,12 @@ tasks:
       doLinearize: false
       doCrosstalk: true
       doBrighterFatter: false
+  cpSkyImage:
+    class: lsst.cp.pipe.cpSkyTask.CpSkyImageTask
+    config:
+      # These values roughly split a single raft into 4x4 superpixels.
+      # As noted below, the sizes are in millimeters, and correspond to a
+      # background image of 6144*6144 pixels (6144*0.01=61.44).
+      largeScaleBackground.xSize: 61.44  # in mm
```

Comment:
If these are known not to work (as shown by the figures on the ticket), then we should file a ticket to figure out correct values, and reference that ticket here and below (for just the cameras that need more development work).

Reply:
But this one shouldn't be larger than the available pixels, probably.

Reply:
I've filed ticket DM-43321 to capture future effort on this, and left a note in each non-HSC YAML referencing this ticket.

All 6k superpixel configs have been reverted to their prior 4k dimensions on this ticket.